recipes/traceback-with-variables/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Since you updated the version and the license is in the source distribution, you can remove the copy here.